frontend/src/app/components/wp-table/timeline/cells/wp-timeline-cell.ts

```
@@ -138,16 +138,16 @@ export class WorkPackageTimelineCell {
     const wasRendered = this.wpElement !== null && body.contains(this.wpElement);
 
     // If already rendered with correct shape, ignore
-    if (wasRendered && (this.elementShape === renderer.type)) {
+    if (wasRendered && this.elementShape === renderer.type) {
       return Promise.resolve();
     }
 
     // Remove the element first if we're redrawing
-    this.clear();
+    !isDuplicatedCell && this.clear();
 
     // Render the given element
     this.wpElement = renderer.render(renderInfo);
-    this.labels = renderer.createAndAddLabels(renderInfo, this.wpElement);
+    this.labels = renderer.createAndAddLabels(renderInfo, this.wpElement, withCustomLabels);
```

This only works because JS will silently discard every parameter that is not matched by the signature, in this case `withCustomLabels` if the renderer is a `TimelineCellRenderer` and not a TimelineMilestoneCellRenderer`. The `createAndAddLabels` method on the superclass `TimelineCellRenderer` does not take 3 parameters but only 2. I am surprised that TypeScript does not balk here.  
It might be an option, also for other methods, to include the additional parameters into the `RenderInfo` which already seems to hold some flags used for rendering.